src/project/qml/MuseScore/Project/AudioGenerationSettings.qml

```diff
@@ -146,23 +177,31 @@ RadioButtonGroup {
             }
 
             IncrementalPropertyControl {
-                width: 96
+                Layout.preferredWidth: numberOfSavesItem.textPart2InSpinbox ? 96 : 60
 
                 minValue: 2
                 maxValue: 30
                 currentValue: settingsModel.numberOfSaves
-                measureUnitsSymbol: qsTrc("project/save", "Saves")
+                measureUnitsSymbol: numberOfSavesItem.textPart2InSpinbox ? numberOfSavesItem.textPart2 : ""
                 step: 1
                 decimals: 0
 
                 navigation.panel: root.navigationPanel
-                navigation.row: index
+                navigation.row: numberOfSavesItem.index
                 navigation.column: 1
+                navigation.accessible.name: qsTrc("project/save", "Every n saves")
```

Comment:
Maybe write this as `"Every N saves"` in case capitalisation affects pronunciation by screen readers? Also, I was initially confused and thought you meant `"Every %n save(s)"`, so a capital N would help avoid that confusion.